config/webpack.config.prod.js

```diff
@@ -216,6 +216,23 @@ module.exports = {
             ),
             // Note: this won't work without `new ExtractTextPlugin()` in `plugins`.
           },
+          // This loader handles font files
+          {
+            test: [
+              /\.eot$/,
+              /\.svg$/,
+              /\.ttf$/,
+              /\.woff$/
+            ],
+            loader: require.resolve('file-loader'),
```

Comment:
Run `gulp lint --fix`

Reply:
Oops! Hah, thought this was cfgov-refresh.

Reply:
what's `gulp`? 🤣

Reply:
Should this repo be aligned with the eslint config in cfgov-refresh or vice versa?